File: Src/java/engine/src/main/java/org/opencds/cqf/cql/engine/elm/visiting/TupleEvaluator.java
package org.opencds.cqf.cql.engine.elm.visiting;

import org.opencds.cqf.cql.engine.execution.State;

import java.util.LinkedHashMap;

public class TupleEvaluator {

  public static Object internalEvaluate(LinkedHashMap<String, Object> ret, State state) {
    return new org.opencds.cqf.cql.engine.runtime.Tuple(state).withElements(ret);
  }
}

File: Src/java/engine/src/commonMain/kotlin/org/opencds/cqf/cql/engine/elm/executing/TupleEvaluator.kt
package org.opencds.cqf.cql.engine.elm.executing

import org.opencds.cqf.cql.engine.runtime.Tuple
import org.opencds.cqf.cql.engine.runtime.Value

object TupleEvaluator {
    fun internalEvaluate(ret: MutableMap<kotlin.String, Value?>?): Tuple {
        return Tuple().withElements(ret!!)
    }
}
